modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxLocalStateAdapter.java

```diff
@@ -17,7 +17,9 @@
 
 package org.apache.ignite.internal.processors.cache.transactions;
 
+import org.apache.ignite.IgniteCheckedException;
```

Comment:
I think this changes doesn't belong to task. Please, remove it.

Reply:
Yeah, some leftovers from squash.